### DISCUSSION TITLE
div class from giscuss gets cut off by material for mkdocs(?)

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#3498] [Q&A] [answered]
Just noticed that a pop-up bubble from Giscus, which tells you to log in in order to react is cut off.

Unfortunately can I not really find out if that is an issue on Giscus' end or if Material for MkDocs does somehow do this.

I do use a custom CSS for giscus, but other than setting the theme to dark and putting the text box at the top is there no difference to the available themes it uses.
CSS can be found [here](https://github.com/Andre601/blog/blob/master/docs/assets/css/style/giscus.css)

Here's an example of what happens:
![image](https://user-images.githubusercontent.com/11576465/150772537-04d2518f-741a-46a8-8f54-7effd9950951.png)

I hope someone can help debug this.
The page where you can see this behaviour in question is [this one here](https://www.andre601.ch/blog/posts/2021/10-20-lets-talk-about-verification/).

I report this to Giscus itself too.

**Comment by squidfunk** [comment, marked as answer]
This must be fixed by Giscus, not by us. In essence, the `iframe` height is incorrectly computed.

**Reply by marcus-d19840604** [reply]
getURL()

**Reply by Andre601** [reply]
Giscus dev mentioned that this is a unwanted/sad byproduct of having the comment box at the top and the comments afterwards.
They proposed a fix which I will add.
